app/controllers/assessments_controller.rb

```diff
@@ -59,7 +59,7 @@ def assessment
     @assessment ||= Assessment.includes(
       { employments: :employment_payments },
       :proceeding_types,
-      gross_income_summary: [
+      applicant_gross_income_summary: [
```

Comment:
What do you think if including `partner_gross_income_summary` to avoid N+1 ?

Reply:
this is the v5 controller (that is in the process of being deleted). V6 doesn't need eager loads because the creations happen in-process